zsh-plugins/boomwaffle-pi/boomwaffle-pi.zsh

```diff
@@ -0,0 +1,7 @@
+
+export PATH="${PATH}:~/code/boom-waffle/bin"
+
+# Add your boomwaffle ssh key to the ssh agent
+boomwaffle-ssh() {
+  eval $(~/code/boom-waffle/bin/add-ssh.sh)
+}
```

Comment:
Ah interesting. Does this assume my local dev is using `~/code/boom-waffle` as my dir? Mine is somewhere else but I can move it :)

Reply:
we can make this an environment variable.  this script is still in the early stages